generator/uscore/templates/access_verify_sequence.rb.erb

```diff
@@ -115,8 +115,11 @@ module Inferno
       end
 
       def scope_granting_access(resource, scopes)
-        scopes.split(' ').find do |scope| 
-          ['patient/*.read', 'patient/*.*', "patient/#{resource}.read", "patient/#{resource}.*"].include? scope
+        non_patient_compartment_resources = ['Encounter', 'Device', 'Location', 'Medication', 'Organization',
+          'Practitioner', 'PractitionerRole', 'RelatedPerson']
+        scopes.split(' ').find do |scope|
+          return true if non_patient_compartment_resources.include?(resource) && ["user/#{resource}.read", "user/#{resource}.*"].include?(scope)
+          ['patient/*.read', 'patient/*.*', "patient/#{resource}.read", "patient/#{resource}.*"].include?(scope)
```

Comment:
Ah, good catch, I find it surprising that this happened -- this must be the first time we've regenerated tests since I made this update.  I looked through the code and it's a situation where `non_patient_compartment_resources` elements will never be passed into this method in the restricted sequence, so there should be no effect to having this code in here (except for generation to not clobber the unrestricted sequence, where we need it).